metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -282,19 +290,37 @@ def _convert_value(param: parameters.Parameter):
             raise CommandException("Please specify --pipeline-path")
 
         pipeline_path = flow.create_kfp_pipeline_yaml(pipeline_path)
-        obj.echo(
-            "\nDone converting *{name}* to {path}".format(
-                name=current.flow_name, path=pipeline_path
+        if yaml_format == "kfp":
+            pass  # KFP pipeline yaml is created by default
+        elif yaml_format == "argo-workflow":
+            flow.update_kfp_yaml(
+                pipeline_path,
+                [("spec.serviceAccountName", "default-editor")],
```

Comment:
Until we set default service account through Argo WorkflowController, this should be `default-editor`.